### DISCUSSION TITLE
Deno 新开发的 JSR，一个新的 JS Registry，支持 TS 并兼容 npm

### DISCUSSION THREAD

**Discussion by giscus** [liruifengv/liruifengv.com#18] [Announcements]
# Deno 新开发的 JSR，一个新的 JS Registry，支持 TS 并兼容 npm

Deno 又搞事情了，开发了一个新的 JS Registry，原生支持 TS，并且兼容 npm

https://liruifengv.com/posts/jsr/

**Comment by yy-zhong** [comment]
> 然后我们创建 deno.json文件，根据文档上说，应该是 jsr.json，这是描述当前包的 metadata 的文件，类似 package.json。但是好像有 bug，执行 deno publish，报错找不到 deno.json。好吧，这里我们就先用 deno.json。

截至2024-05-29，这个问题已经被修复了。现在使用`jsr.json`来配置不会报错。
